src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+export const checkboxDataTestIds = {
+  base: 'checkbox__base',
+};
+
+interface CheckboxProps {
+  isValueChecked: boolean;
+}
+
+const Checkbox = ({ isValueChecked }: CheckboxProps): JSX.Element => {
+  return (
+    <div className="flex text-white" data-test-id={checkboxDataTestIds.base}>
+      <input
+        type="checkbox"
+        id="checkbox"
+        checked={isValueChecked}
+        onChange={() => {
+          // Do nothing
```

Comment:
If it doesn't do anything, should we remove it?

Reply:
@hoangmirs because `eslint` catches the empty function so I put a comment to bypass it 😄